src/main/webapp/css/home_style.css

```diff
@@ -0,0 +1,30 @@
+.heading {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  height: 50vh;
+}
+
+h1 {
+  font: 400 60px Roboto, Arial, sans-serif;
+  margin: 0px 0px;
+}
+
+.logo {
```

Comment:
nit: In your HTML, .logo comes before h1 so it makes more sense to me to have the rule come first as well.